File: SDKs/amazonaws.com/comprehendmedical/2018-10-30/typescript/src/sdk/models/shared/outputdataconfig.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// OutputDataConfig
/** 
 * The output properties for a detection job.
**/
export class OutputDataConfig extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=S3Bucket" })
  s3Bucket: string;

  @SpeakeasyMetadata({ data: "json, name=S3Key" })
  s3Key?: string;
}

File: SDKs/amazonaws.com/comprehendmedical/2018-10-30/python/src/sdk/models/shared/outputdataconfig.py
from dataclasses import dataclass, field
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclass
class OutputDataConfig:
    r"""OutputDataConfig
    The output properties for a detection job.
    """
    
    s3_bucket: str = field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('S3Bucket') }})
    s3_key: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('S3Key') }})
    
